corehq/apps/cleanup/tasks.py

```diff
@@ -24,14 +24,18 @@
 logger = logging.getLogger(__name__)
 
 
-@periodic_task(run_every=crontab(minute=0, hour=0), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
+@periodic_task(queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
```

Comment:
Using the `periodic_task` decorator without specifying `run_every` causes [this error](https://dimagi.sentry.io/issues/4723801186/?environment=staging&project=136860&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=24h&stream_index=0) and prevents celery from properly starting up. It is probably easiest to just remove the decorator entirely, but I'd also vote for doing that on a separate PR to facilitate pushing out that change asap.

Reply:
ahh good call